.travis.yml

```diff
@@ -15,12 +15,15 @@ install:
   - bundle install --retry=3
 
 env:
-  - "RAILS_VERSION=4.0"
-  - "RAILS_VERSION=4.1"
-  - "RAILS_VERSION=4.2"
-  - "RAILS_VERSION=master"
+  global:
+    - JRUBY_OPTS=-Xcext.enabled=true
```

Comment:
housekeeping: I don't think is a good idea to set JRUBY_OPTS this way since [travis uses `JRUBY_OPTS="--server -Dcext.enabled=false -Xcompile.invokedynamic=false"` now](http://docs.travis-ci.com/user/environment-variables/#Default-Environment-Variables) ([cext.enabled=false is the future](http://docs.travis-ci.com/user/languages/ruby/#JRuby%3A-C-extensions-support-is-disabled)) and ref: https://github.com/rspec/rspec-dev/pull/115